electrum_nmc/electrum/gui/qt/configure_dns_dialog.py

```diff
@@ -189,6 +189,7 @@ def create_address_record(self):
             "IPv4": "ip4",
             "IPv6": "ip6",
             "Tor": "tor",
+            "IPFS": "ipfs",
```

Comment:
I'd prefer that IPFS be positioned between Freenet and ZeroNet. (Same goes for everywhere else in this PR.) The intended ordering is: IP addresses, anonymity transport networks (Tor/I2P), anonymity content hosting networks (Freenet), non-anonymity content hosting networks (IPFS/ZeroNet).